examples/access-controllable-examples/README.md

```diff
@@ -0,0 +1,450 @@
+
+#[near_bindgen]
+impl Counter {
+   /// In the constructor we set up a super admin,
+   /// which can control the member lists of all user groups
+   #[init]
+   pub fn new() -> Self {
+      let mut contract: Counter = Self{
+         counter: 0,
+         __acl: __Acl::default(),
+      };
+
+      contract.acl_init_super_admin(near_sdk::env::predecessor_account_id());
+
+      contract
+   }
+
+   /// unprotected function, every one can call this function
+   pub fn unprotected(&mut self) {
+      self.counter += 1;
+   }
+
+   /// only the users from GroupA can call this method
+   #[access_control_any(roles(UsersGroups::GroupA))]
+   pub fn level_a_incr(&mut self) {
+      self.counter += 1;
+   }
+
+   /// only the users from GroupA or GroupB can call this method
+   #[access_control_any(roles(UsersGroups::GroupA, UsersGroups::GroupB))]
+   pub fn level_ab_incr(&mut self) {
+      self.counter += 1;
+   }
+
+
+   /// view method for get current counter value, every one can use it
+   pub fn get_counter(&self) -> u64 {
+      self.counter
+   }
+}
+```
+
+## The contract methods description
+### acl_storage_prefix
+`acl_storage_prefix` is a method that returns the common prefix of keys for storing the members and the admins of groups.
+`__acl` by default.
+
+```shell
+$ near call <CONTRACT_ACCOUNT> acl_storage_prefix --accountId <ALICE_ACCOUNT>
+Scheduling a call: <CONTRACT_ACCOUNT>.acl_storage_prefix()
+Doing account.functionCall()
+Transaction Id <TRANSACTION_ID>
+To see the transaction in the transaction explorer, please open this url in your browser
+https://explorer.testnet.near.org/transactions/<TRANSACTION_ID>
+[ 95, 95, 97, 99, 108 ]
+$ python3
+>>> print(' '.join(str(b) for b in bytes("__acl", 'utf8')))
+95 95 97 99 108
+```
+
+Example of changing acl storage prefix key:
+```rust
+#[near_bindgen]
+#[access_control(role_type(UsersGroups), storage_prefix = "__custom_prefix")]
+#[derive(Default, BorshSerialize, BorshDeserialize)]
+struct Counter {
+  counter: u64,
+}
+```
+
+### acl_init_super_admin
+`acl_init_super_admin` is a method that adds `account_id` as a super-admin _without_ checking any permissions
+in case there are no super-admins. Do nothing if at least one super-admin exists. This function can be used to add a super-admin during contract initialization. 
+Moreover, it may provide a recovery mechanism if (mistakenly) all super-admins have been removed. 
+
+**Return value:** the return value indicates whether `account_id` was added as super-admin.
+
+It is `#[private]` in the implementation provided by this trait, i.e. only the contract itself may call this method.
+
+```shell
+$ near call <CONTRACT_ACCOUNT> acl_init_super_admin '{"account_id": "<SUPER_ADMIN_ACCOUNT>"}' --accountId <CONTRACT_ACCOUNT>
+true
+```
+
+If the method succeeds, the following event will be emitted:
+```json
+{
+   "standard":"AccessControllable",
+   "version":"1.0.0",
+   "event":"super_admin_added",
+   "data":{
+      "account":"test.near",
```

Comment:
```suggestion
      "account":"<SUPER_ADMIN_ACCOUNT",
```

Reply:
fix https://github.com/aurora-is-near/near-plugins/pull/34/commits/1a7a34b88d2b4513d1de096b408ef5fc17e46209